src/Util.h

```diff
@@ -265,6 +265,19 @@ void dir_rmdir(const std::string &name);
 /** Wrapper for stat(). Asserts upon error. */
 FileStat file_stat(const std::string &name);
 
+/** Read the entire contents of a file into a vector<char>. The file
+ * is read in binary mode. Errors trigger an assertion failure. */
+std::vector<char> read_entire(const std::string &pathname);
```

Comment:
I think these should be named `read_entire_file` and `write_entire_file`, to make the code using it easier to read, and to be consistent with the rest of the file helper functions here.

Reply:
Agreed.